packages/babel-parser/test/fixtures/typescript/types/infer-with-constraints/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "BABEL_8_BREAKING": false
```

Comment:
Is this behavior different in Babel 8? If it is, can you also add a Babel 8 test?

Reply:
In babel 8, the AST for type parameters will be changed. I'll add tests for babel 8 later!

Reply:
[b129eef](https://github.com/babel/babel/pull/14476/commits/b129eef681b641cabb8752f88af993044f601cec) :+1: